allauthdemo/templates/account/email/!email_confirmation_subject.html

```diff
@@ -0,0 +1 @@
+Confirm your email
```

Comment:
Translation block required

Reply:
Sorry, that was a temporary file, deleted. ```demo/templates/account/email/email_confirmation_subject.txt``` has translation block.